server/index.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const path = require('path');
+const app = express();
+const port = process.env.PORT || 3003;
+// console.log(__dirname);
+app.use(express.static(path.join(__dirname, '/../public')));
+
```

Comment:
Everything looks pretty good just don't forget to bring in some middleware to parse the req body

Reply:
Thanks! It's on the to-do list.